package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "tree-browser-bundle",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://secure.travis-ci.org/symfony-cmf/TreeBrowserBundle.png?branch=master)](http://travis-ci.org/symfony-cmf/TreeBrowserBundle) [![Latest Stable Version](https://poser.pugx.org/symfony-cmf/tree-browser-bundle/version.png)](https://packagist.org/packages/symfony-cmf/tree-browser-bundle) [![Total Downloads](https://poser.pugx.org/symfony-cmf/tree-browser-bundle/d/total.png)](https://packagist.org/packages/symfony-cmf/tree-browser-bundle)",
```

Comment:
should this not be a human readable description of what this bundle provides (in terms of a bower package)?